src/abstractions/software/assembler/mockData/index.js

```diff
@@ -1,3 +1,4 @@
 import symbolLess from './symbolLess'
```

Comment:
also change references in the test file
```suggestion
import symbolLessAssembly from './symbolLessAssembly'
```